src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+const RENT_PRICE = 40;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const MEDIUM_TERM = 3;
+const MEDIUM_TERM_DISCOUNT = 20;
+
 function calculateRentalCost(days) {
-  // write code here
+  const numberOfDays = days;
+  const basePrice = numberOfDays * RENT_PRICE;
+
+  if (numberOfDays >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  }
+
+  if (numberOfDays < LONG_TERM && numberOfDays >= MEDIUM_TERM) {
```

Comment:
The check `numberOfDays < LONG_TERM` is unnecessary because if the code execution reaches this line, it means the condition in the previous `if` statement (`numberOfDays >= LONG_TERM`) was false, making this check redundant. You can remove this condition to simplify the logic.